contracts/src/v0.8/dev/automation/2_1/KeeperRegistryLogicA2_1.sol

```diff
@@ -87,15 +107,7 @@ contract KeeperRegistryLogicA2_1 is
         return (false, bytes(""), UpkeepFailureReason.PERFORM_DATA_EXCEEDS_LIMIT, gasUsed, fastGasWei, linkNative);
     }
 
-    performData = abi.encode(
-      PerformDataWrapper({
-        checkBlockNumber: uint32(_blockNum() - 1),
```

Comment:
are we going to get block number and hash from core node side now?

Reply:
yeah that's the plan! I think it keeps the contract interface much simpler